js/src/features/project/components/ProjectTeamsList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { List } from "antd";
+
+import { Project } from "ProjectModels";
+import { ProjectTeamsListItem } from "./ProjectTeamsListItem";
+
+interface Props {
+  project: Project;
```

Comment:
you need only teams and slug props. Not entire project

Reply:
done